console/src/domain/dataset/sdk.ts

```diff
@@ -95,19 +95,15 @@ export class DatasetObject {
 
         // @ts-ignore
         Object.entries(data).forEach(([key, value]: [string, any]) => {
-            if (key !== 'annotations') return
-
-            try {
-                Object.entries(value).forEach(([k, v]: [string, any]) => {
-                    if (typeof v === 'number' || typeof v === 'string' || typeof v === 'boolean') {
-                        this.summary[k] = v
-                    }
-                })
-                value.annotations.forEach((item?: any) => this.setProps(item))
-                this.setProps(value.mask)
-            } catch (e) {
-                // eslint-disable-next-line no-console
-                console.error(e)
+            if (!key.startsWith('annotation/')) return
+            if (typeof value === 'number' || typeof value === 'string' || typeof value === 'boolean') {
+                this.summary[key] = value
+            }
+            if (key === 'annotation/annotations') {
```

Comment:
`annotation/annotations` is a special flag or a typo?

Reply:
Neither. Like `annotation/mask` this is currentlly hard coded with the `pfp` dataset. 
https://github.com/star-whale/starwhale/blob/8614448d72ce07643a64179bed44ba2a8f4e6485/example/PennFudanPed/pfp/dataset.py#L31-L50